djangoproject/templates/blog/blog_pagination.html

```diff
@@ -1,19 +1,23 @@
+{% load i18n %}
+
 {% if is_paginated %}
   <div class="pagination">
     <ul class="nav-pagination" role="navigation">
       {% if page_obj.has_previous %}
         <li><a class="previous" href="?page={{ page_obj.previous_page_number }}">
           <i class="icon icon-chevron-left"></i>
-          <span class="visuallyhidden">Previous</span>
+          <span class="visuallyhidden">{% translate "Previous" %}</span>
```

Comment:
I would add a contextual marker here:
```suggestion
          <span class="visuallyhidden">{% translate "Previous" context "pagination" %}</span>
```

Reply:
@bmispelon Yeah, good idea - I don't use context enough with translation!!